src/iterative_sorting/iterative_sorting.py

```diff
@@ -10,16 +10,16 @@ def selection_sort( arr ):
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
         #print(arr[smallest_index], smallest_index, cur_index) 
-        while cur_index < len(arr) - 1:
+        for n in  range(cur_index, len(arr)):
```

Comment:
For selection sort, the current index should be compared to any indexes to the right of it. Therefore, your for loop should be like this:

```
for n in  range(cur_index + 1, len(arr)):
```